### ISSUE TITLE
ampersand sign converted into unicode when added to analytics config.

### ISSUE THREAD

**Issue by milindmore22** [ampproject/amp-toolbox-php#529]
**Bug Description** 

A user reported on support forums that the `&` is being converted to Unicode character `\u0026` when being added to the analytics config.
 
 Disabling optimizer it did output correctly 
`add_filter( 'amp_enable_optimizer', '__return_false' );` 

**Steps to reproduce**
You can reproduce using the AMP plugin for WordPress, steps assuming that you have WordPress site with AMP plugin installed

Step 1 : Login as Admin
Step 2: Goto AMP->Settings->Analytics
Step 3: Enter the type as "test" (you can add anything as type was unknow from support topics)
Step 4: add JSON config as below

```json 
{
  "requests": {
    "custom_params": "cp1=test&cp2=test"
  },
  "vars": {
    "env": "-preproduction"
  },
  "triggers": {
    "trackConsent": {
      "on": "visible",
      "request": "impression"
    }
  }
}
```

**Expected Output** 
Same output as entered in config.

**Current Behaviour**

```json 
{
  "requests": {
    "custom_params": "cp1=test\u0026cp2=test"
  },
  "vars": {
    "env": "-preproduction"
  },
  "triggers": {
    "trackConsent": {
      "on": "visible",
      "request": "impression"
    }
  }
}
```

**Screenshot**
| Input  | Output |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/1132541/184157996-86013f4a-b612-4e99-a404-071d498b236c.png) | ![image](https://user-images.githubusercontent.com/1132541/184158248-9eb53ff4-fbc1-4912-a365-51fa2a3929d8.png) |

**Comment by westonruter** [comment]
I don't think this is a problem. In JavaScript, `\u0026` is the encoding of an ampersand the same way that `&amp;` or `&#x26;` is the way to encode the ampersand in HTML and `\0026` in CSS. They all get decoded in the end as `&`.
